src/scripts/achievements/AchievementHandler.ts

```diff
@@ -311,28 +351,65 @@ class AchievementHandler {
+
+        AchievementHandler.addAchievement('Some nice help for the Day Care', 'Unlock 5 Hatchery Helpers', new HatcheryHelperRequirement(5, 0), 0.1);
+        AchievementHandler.addAchievement('Why do they have to work in shifts?', 'Unlock 10 Hatchery Helpers', new HatcheryHelperRequirement(10, 0), 0.3);
+        AchievementHandler.addAchievement('My loyal helpers', 'Get 3 Hatchery Helpers to 10% bonus efficiency', new HatcheryHelperRequirement(3, 10), 0.4);
+        AchievementHandler.addAchievement('Let\'s try some other helpers too?', 'Get 5 Hatchery Helpers to 10% bonus efficiency', new HatcheryHelperRequirement(5, 10), 0.5);
+        AchievementHandler.addAchievement('Sam just wants to help', 'Get 10 Hatchery Helpers to 10% bonus efficiency', new HatcheryHelperRequirement(10, 10), 1);
+        AchievementHandler.addAchievement('When are you going to breed yourself?', 'Get 10 Hatchery Helpers to 25% bonus efficiency', new HatcheryHelperRequirement(10, 25), 1.3);
+
+        AchievementHandler.addAchievement('Getting some help at the farm', 'Unlock 3 Farm Hands', new FarmHandRequirement(3), 0.2);
+        AchievementHandler.addAchievement('Starting a farming empire', 'Unlock 6 Farm Hands', new FarmHandRequirement(6), 0.3);
+        AchievementHandler.addAchievement('Will I ever need all this help?', 'Unlock 9 Farm Hands', new FarmHandRequirement(9), 0.4);
+
+        AchievementHandler.addAchievement('I hope it\'s not dangerous', 'Infect 20 Pokémon with Pokerus', new PokerusStatusRequirement(20, GameConstants.Pokerus.Infected), 0.1);
+        AchievementHandler.addAchievement('Anyway to get rid of it?', 'Infect 50 Pokémon with Pokerus', new PokerusStatusRequirement(50, GameConstants.Pokerus.Infected), 0.2);
+        AchievementHandler.addAchievement('It is the flu season', 'Infect 100 Pokémon with Pokerus', new PokerusStatusRequirement(100, GameConstants.Pokerus.Infected), 0.3);
+        AchievementHandler.addAchievement('Where is nursery Joy?', 'Infect 250 Pokémon with Pokerus', new PokerusStatusRequirement(250, GameConstants.Pokerus.Infected), 0.4);
+        AchievementHandler.addAchievement('Infecting the whole world', 'Infect 1000 Pokémon with Pokerus', new PokerusStatusRequirement(1000, GameConstants.Pokerus.Infected), 0.5);
+        AchievementHandler.addAchievement('Curing them one at a time', 'Have 10 Resistant Pokémon', new PokerusStatusRequirement(10, GameConstants.Pokerus.Resistant), 0.3);
+        AchievementHandler.addAchievement('Pokémon center regular', 'Have 50 Resistant Pokémon', new PokerusStatusRequirement(50, GameConstants.Pokerus.Resistant), 0.5);
+        AchievementHandler.addAchievement('Doctor in training', 'Have 250 Resistant Pokémon', new PokerusStatusRequirement(250, GameConstants.Pokerus.Resistant), 1);
+        AchievementHandler.addAchievement('I should open my own Pokémon Center', 'Have 500 Resistant Pokémon', new PokerusStatusRequirement(500, GameConstants.Pokerus.Resistant), 1.5);
+
```

Comment:
'Pokerus' -> 'Pokérus', and 'Have x Resistant Pokemon' -> 'Have x Pokemon Resistant to Pokerus'

```suggestion
        AchievementHandler.addAchievement('I hope it\'s not dangerous', 'Infect 20 Pokémon with Pokérus', new PokerusStatusRequirement(20, GameConstants.Pokerus.Infected), 0.1);
        AchievementHandler.addAchievement('Anyway to get rid of it?', 'Infect 50 Pokémon with Pokérus', new PokerusStatusRequirement(50, GameConstants.Pokerus.Infected), 0.2);
        AchievementHandler.addAchievement('It is the flu season', 'Infect 100 Pokémon with Pokérus', new PokerusStatusRequirement(100, GameConstants.Pokerus.Infected), 0.3);
        AchievementHandler.addAchievement('Where is nursery Joy?', 'Infect 250 Pokémon with Pokérus', new PokerusStatusRequirement(250, GameConstants.Pokerus.Infected), 0.4);
        AchievementHandler.addAchievement('Infecting the whole world', 'Infect 1000 Pokémon with Pokérus', new PokerusStatusRequirement(1000, GameConstants.Pokerus.Infected), 0.5);
        AchievementHandler.addAchievement('Curing them one at a time', 'Have 10 Pokémon Resistant to Pokérus', new PokerusStatusRequirement(10, GameConstants.Pokerus.Resistant), 0.3);
        AchievementHandler.addAchievement('Pokémon center regular', 'Have 50 Pokémon Resistant to Pokérus', new PokerusStatusRequirement(50, GameConstants.Pokerus.Resistant), 0.5);
        AchievementHandler.addAchievement('Doctor in training', 'Have 250 Pokémon Resistant to Pokérus', new PokerusStatusRequirement(250, GameConstants.Pokerus.Resistant), 1);
        AchievementHandler.addAchievement('I should open my own Pokémon Center', 'Have 500 Pokémon Resistant to Pokérus', new PokerusStatusRequirement(500, GameConstants.Pokerus.Resistant), 1.5);

```